### ISSUE TITLE
Not able to download outdoor_day2

### ISSUE THREAD

**Issue by hamidehkerdegari** [daniilidis-group/EV-FlowNet#11]
I am not able to download outdoor_day2 data from the google drive link that you provided. Is there any other link to this data please?

Thanks

**Comment by alexzzhu** [comment]
Are you talking about the ROS bags? We noticed an issue with them a few days ago, and reuploaded those files on March 30. If this is the case, could you check to see if they're there now?

**Comment by hamidehkerdegari** [comment]
No, I mean the converted pngs and TFRecords. Since outdoor_day2 is a huge file, I need another way to easily download it. At the moment, this download gets failed through google drive download.

**Comment by alexzzhu** [comment]
What is the error that you see when downloading this file? Things seem to be working on my end. Currently Google Drive is the best/only method for us to distribute data.

**Comment by hamidehkerdegari** [comment]
I can download all the indoor and outdoor files except outdoor_day2 that is quite heavy to be downloaded through google drive.

**Comment by alexzzhu** [comment]
Is the issue that the file is too large? I can try compress/shard it and reupload in that case.

**Comment by hamidehkerdegari** [comment]
yes, that's the issue. That would be great. Thanks

**Comment by alexzzhu** [comment]
Ok, no problem! I'll let you know when it's uploaded.

**Comment by alexzzhu** [comment]
If you check the google drive folder, there should be a subfolder named 'parts'. This contains the original tfrecord file split into zip shards, which you should be able to reconstruct once you download them all. Please let me know if this resolves your issue.

https://drive.google.com/drive/folders/147pYsv2QWWmFmGjoM5bRr6JOYXYoJEin

**Comment by hamidehkerdegari** [comment]
Thanks for uploading the parts folder. Is it also possible to zip the png files the same as tfrecords and upload them on google drive?
